src/components/common/FullViewSnapshotButton.tsx

```diff
@@ -155,7 +178,36 @@ function FullViewSnapshotButton({
             }
             value={selectedBillingProfile?.profileName || ''}
           />
-          <Typography>If this is the correct billing project - just click select</Typography>
+          <div style={{ marginTop: 8 }}>
+            <FormLabel
+              sx={{ fontWeight: 600, color: 'black' }}
+              htmlFor="authorization-domain-select"
+            >
+              Authorization Domain
+              <span style={{ fontWeight: 400, color: 'black' }}> - (optional)</span>
```

Comment:
Is the use of `(optional)` consistent with other optional vs required inputs in TDR UI? I didn't see it used elsewhere in the code.